docs/features/techdocs/using-cloud-storage.md

```diff
@@ -93,3 +93,177 @@ techdocs:
 
 Your Backstage app is now ready to use Google Cloud Storage for TechDocs, to
 store the static generated documentation files.
+
+## Configuring AWS S3 Bucket with TechDocs
+
+Follow the
+[official AWS S3 documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)
+for the latest instructions on the following steps involving AWS S3.
+
+**1. Set `techdocs.publisher.type` config in your `app-config.yaml`**
+
+Set `techdocs.publisher.type` to `'awsS3'`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'awsS3'
+```
+
+**2. AWS Policies**
+
+AWS Policies lets you **control access** to Amazon Web Services (AWS) products
+and resources.  
+Here we will use a user policy **and** a bucket policy to show you the different
+possibilities you have but you can use only one.
+
+<img data-zoomable src="../../assets/techdocs/aws-s3.png" alt="AWS S3" />
+
+This is an example of how you can manage your policies:
+
+a. Admin user creates a **bucket policy** granting a set of permissions to our
+TechDocs user.
+
+b. Admin user attaches a **user policy** to the TechDocs user granting
+additional permissions.
+
+c. TechDocs User then tries permissions granted via both the **bucket** policy
+and the **user** policy.
+
+**2.1 Creation**
+
+**2.1.1 Create an Admin user** (if you don't have one yet)
+
+Create an **administrator user** account `ADMIN_USER` and grant it administrator
+privileges by attaching a user policy giving the account **full access**.  
+Note down the Admin User credentials and IAM User Sign-In URL as you will need
+to use this information in the next step.
+
+**2.1.2 Create an AWS S3 Bucket**
+
+Using the credentials of your Admin User `ADMIN_USER`, and the special IAM user
+sign-in URL, create a dedicated **bucket** for TechDocs sites. techdocs-backend
+will publish documentation to this bucket. TechDocs will fetch files from here
+to serve documentation in Backstage.
+
+Set the name of the bucket to `techdocs.publisher.awsS3.bucketName`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'awsS3'
+    awsS3:
+      credentials:
+        $file: '/path/to/aws_application_credentials.json'
```

Comment:
This needs to be either removed or updated with `accessKeyId` and `secretAccessKey` as described below.